snooty.toml

```diff
@@ -2,10 +2,15 @@ name = "kotlin-sync"
 title = "Kotlin Sync Driver"
 
 intersphinx = [ "https://www.mongodb.com/docs/manual/objects.inv",
+                "https://www.mongodb.com/docs/drivers/objects.inv",
                 "https://www.mongodb.com/docs/atlas/objects.inv"
               ]
 
-# toc_landing_pages = ["/paths/to/pages/that/have/nested/content"]
+toc_landing_pages = []
 
-# [constants]
-# constant = "value"
+[constants]
+driver-long = "MongoDB Kotlin Sync Driver"
+driver-short = "Kotlin Sync Driver"
+language = "Kotlin"
+version-number = "5.1"
+version = "v{+version-number+}"
```

Comment:
C: Normally there is also a full version that is: "{+version+}.0" for the specific version like 15.9.0 for example. Other repos also do not have v{+version-number+} for version, and just have one version-number. Will defer to Rea about proper constants though!

Reply:
we could do both, idk when we might use the version constant but full-version is definitely helpful in places

Reply:
For context, I noticed a handful of repos have a constant to represent `v{+version-number+}`.